strategies/affected/src/main/java/org/arquillian/smart/testing/strategies/affected/ClassDependenciesGraph.java

```diff
@@ -86,13 +92,28 @@ void buildTestDependencyGraph(Collection<File> testJavaFiles) {
             JavaClass testJavaClass = builder.getClassDescription(changedTestClassNames);
             if (testJavaClass != null) {
                 final String[] imports = testJavaClass.getImports();
+
                 final List<String> manualProductionClasses = calculateManuallyAddedDependencies(testJavaClass);
                 manualProductionClasses.addAll(Arrays.asList(imports));
-                addToIndex(new JavaElement(testJavaClass), manualProductionClasses);
+
+                final List<Path> files = calculateWatchedFiles(testJavaClass);
+
+                addToIndex(new JavaElement(testJavaClass), manualProductionClasses, files);
             }
         }
     }
 
+    private List<Path> calculateWatchedFiles(JavaClass testJavaClass) {
```

Comment:
Isn't it rather `resolve` or even `find` than `calculate`? Same for method about `calculateManuallyAddedDependencies` - we don't really do any calculation by the pure definition of this word I believe.